### ISSUE TITLE
Card Description

### ISSUE THREAD

**Issue by NahuelGil98** [altimetrik-onboarding-uy/ngil#4]

**Comment by NahuelGil98** [comment]
As I user I want that when I clicking on a card appears the card info So that I can edit its title and description.
- Inside the modal must be a save button and a close button that cancels the action.

**Comment by stefaniaoverman** [comment]
non-technical users probably wont know what is a "modal"...maybe you could explain it with another words.
